features/nanostack/coap-service/source/coap_security_handler.c

```diff
@@ -156,7 +180,11 @@ static void coap_security_handler_reset(coap_security_t *sec)
 #endif
 
     mbedtls_entropy_free(&sec->_entropy);
-    mbedtls_ctr_drbg_free(&sec->_ctr_drbg);
+#if defined(MBEDTLS_CTR_DRBG_C)
+    mbedtls_ctr_drbg_free(&sec->_drbg);
+#elif defined(MBEDTLS_HMAC_DRBG_C)
+    mbedtls_hmac_drbg_free(&sec->_drbg);
+#endif
```

Comment:
Similarly here #else ?

Reply:
No, we don't want to pollute code with dead code